### ISSUE TITLE
queue:work & Duplicates

### ISSUE THREAD

**Issue by netspear** [freescout-helpdesk/freescout#2422]
I have Freescout on a ubuntu VM.

Web Server | nginx/1.18.0
PHP 8.1.2-1ubuntu2.9

I'm starting to have an issue on and off for the past 2 weeks or so.

The queue:work would show "Last run: ? Last successful run: ? Try to clear cache to force command to start."
and there would be a list of jobs in the Queued Jobs that would sometime be over 30 mins old or even longer.

I could clear the cache and run it and get it to start working again. but it happens again every now and then, and a lot of times we don't realize it for a while and it really interrupts workflow.

another issue, I can send an email from FreeScout and it sends, but still shows in the Queued Jobs as an attempt and keeps sending the email again and again.

Would anyone be able to help me resolve these issues? I would really appreciate it.

**Comment by freescout-helpdesk** [comment]
Is it similar to https://github.com/freescout-helpdesk/freescout/issues/2428 ?

**Comment by netspear** [comment]
No. but I think I found the issue.

I found this error in the App Logs:
`error | production | 2022-11-30 13:12:35 | Could not save outgoing reply to the IMAP folder, IMAP folder not found: Sent Items - Error: fread(): SSL: Connection reset by peer; File: /var/www/html/vendor/webklex/php-imap/src/Connection/Protocols/ImapProtocol.php (108)`

I found that in the connection settings of the mailbox defined "IMAP Folder To Save Outgoing Replies" and i had set it as "Sent Items" while it's the same name as my sent folder, I'm guessing it had issues saving to it.
So I removed that, and so far, this issue hasn't happened again.
if it comes up again, I'll let you know.
